src/lib/components/Widget.js

```diff
@@ -72,8 +72,9 @@ export const Widget = React.forwardRef((props, forwardedRef) => {
   const ethersProviderContext = useContext(EthersProviderContext);
 
   const cache = useCache();
-  const near = useNear();
-  const accountId = useAccountId();
+  const chainId = configs && configs.findLast(config => config && config.chainId)?.chainId;
```

Comment:
Maybe we should name it `networkId` to be consistent with other places

Reply:
I agree, updated